src/scout_apm/celery.py

```diff
@@ -7,20 +7,21 @@
 from scout_apm.core.tracked_request import TrackedRequest
 
 
-# TODO: Capture queue.
-# https://stackoverflow.com/questions/22385297/how-to-get-the-queue-in-which-a-task-was-run-celery?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa
-def prerun_callback(sender=None, headers=None, body=None, **kwargs):
-    name = kwargs["task"].name
+def prerun_callback(task=None, **kwargs):
+    tracked_request = TrackedRequest.instance()
+    tracked_request.mark_real_request()
 
-    tr = TrackedRequest.instance()
-    tr.mark_real_request()
-    span = tr.start_span(operation=("Job/" + name))
-    span.tag("queue", "default")
+    span = tracked_request.start_span(operation=("Job/" + task.name))
+    delivery_info = task.request.delivery_info
+    span.tag("is_eager", delivery_info.get("is_eager", False))
+    span.tag("exchange", delivery_info.get("exchange", "unknown"))
+    span.tag("routing_key", delivery_info.get("routing_key", "unknown"))
+    span.tag("queue", delivery_info.get("queue", "unknown"))
```

Comment:
I think the core agent knows how to handle the `queue` tag, but the others are not supported explicitly and will be silently dropped, IIRC.  Are we wanting to track this info in span tags or in context? @cschneid might chime in here.

Reply:
Aha, I didn't know that span tags are dropped. Moving to request tags/context is probably the right move in that case, but I'll wait for @cschneid 's input.

Reply:
Currently in the core agent, span tags are very specific, and unknown ones are dropped. Some upcoming work will be to support the new trace structure, where arbitrary span tags are sent onward, but that's not for a bit. `queue` is recognized as the string queue name that the job was enqueued on & then popped off of.

For now it's probably ok to add these as request tags (context). But anyway, this does feel like it's scoped to the whole request (the job), not just the section of code that celery is calling first..

Reply:
Moving them all to request tags then. I think I had a concern that celery might recursively execute tasks with its fancy chord structures, but after checking the docs again I don't think that's the case